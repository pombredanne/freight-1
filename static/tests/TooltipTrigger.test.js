import React from 'react';
import renderer from 'react-test-renderer';
import TooltipTrigger from '../components/TooltipTrigger.jsx';
import {shallow} from 'enzyme';
import ReactTestUtils from 'react-dom/test-utils';

describe('<TooltipTrigger />', () => {
  it('should render a snapshot of <TooltipTrigger />', () => {
    const props = {
    "placement": "bottom",
    "title": <div>foo</div>,
    "viewport": {
      "selector": "body",
      "padding": 5
    }
    }

    const appList = [{
      "environments": {
        "production": {
          "defaultRef": "master"
        },
        "staging": {
          "defaultRef": "HEAD"
        },
      },
      "id": "1",
      "name": "freight",
      "repository": "https://github.com/getsentry/freight.git"
    }]


    const shallowRenderer = ReactTestUtils.createRenderer();
    const result = shallowRenderer.render(<TooltipTrigger {...props} children={props.title} />);
    expect(result).toMatchSnapshot();
  })
})
