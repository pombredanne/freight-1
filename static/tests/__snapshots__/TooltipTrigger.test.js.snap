// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TooltipTrigger /> should render a snapshot of <TooltipTrigger /> 1`] = `
<div>
  foo
</div>
`;
